cle/memory.py

```diff
@@ -62,6 +65,20 @@ def unpack_word(self, addr, size=None, signed=False, endness=None):
         :param bool signed: Whether the data should be extracted signed/unsigned. Default unsigned
         :param archinfo.Endness endness: The endian to use in packing/unpacking. Defaults to memory endness
         """
+        if size == 16:
+            # Python doesn't have struct format string to support 16-byte long words
+            # load the word in two 8-byte chunks
+            fmt = self._arch.struct_fmt(size=8, signed=signed, endness=endness)
+            a = self.unpack(addr, fmt)[0]
+            b = self.unpack(addr + 8, fmt)[0]
+            if endness is None:
+                endness = self._arch.memory_endness
+            if endness == archinfo.Endness.BE:
+                return (a << 64) | b
+            elif endness == archinfo.Endness.LE:
+                return (b >> 64) | a
```

Comment:
is this right? won't `b >> 64` always give you 0 or -1?

Reply:
Nice catch! Fixed and added a test case.